packages/indexer-common/src/indexer-management/__tests__/actions.ts

```diff
@@ -365,11 +366,62 @@ describe('Actions', () => {
             type: ActionType.ALLOCATE,
             source: 'indexerAgent',
           },
+          order: {
+            id: Sort.desc,
+          },
         })
         .toPromise(),
     ).resolves.toHaveProperty(
       'data.actions',
-      expecteds.sort((a, b) => (a.id > b.id ? -1 : 1)),
+      expecteds.sort((a, b) => (a.id < b.id ? -1 : 1)),
+    )
+  })
+
+  test('Queue many actions with status and invalid ordering specifications', async () => {
```

Comment:
Also not sure what this is testing just from the name. You can queue actions in an invalid order?

Reply:
No :( That's a bad name, updating to `Queue many actions and retrieve all of a certain status with invalid ordering`, if that's better. This is meant to test for wrong input for `orderBy`

Reply:
sounds good!